tracer/test/Datadog.Trace.BenchmarkDotNet.Tests/Datadog.Trace.BenchmarkDotNet.Tests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
```

Comment:
Not necessary, referenced implicitly
```suggestion
```

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/3774/commits/6325464f33bcf5fa227d094997d59350b1841a49